lib/python/qmk/cli/doctor.py

```diff
@@ -240,6 +266,10 @@ def doctor(cli):
         ok = False
 
     # Make sure the tools are at the correct version
+    check_avrdude_version()
+    check_dfu_util_version()
+    check_dfu_programmer_version()
+
```

Comment:
```suggestion
    for check in (check_arm_gcc_version, check_avr_gcc_version, check_avrdude_version, check_dfu_util_version, check_dfu_programmer_version):
        if not check():
            ok = False
```

I can't make a suggestion against unchanged lines but with this lines 273-278 can be removed.